onboardingCli.js

```diff
@@ -115,52 +238,6 @@ class OnboardingCli {
         }
     }
 
-    /**
-     * Gets either a static URL from the manifest, or asks the onboarder to create one.
-     */
-    getUrl(onboarding, flow, answers) {
-        if (flow.name === 'url') {
-            var replaceVars = this.getReplaceVars(answers);
-            return Promise.resolve(this.replaceVarsInValue(flow.description, replaceVars));
-        } else if (flow.name === 'method') {
-            return onboarding[flow.description](answers);
-        }
-    }
-
-    // this resolves variables in the onboarding flow with dynamic values retreived from the user
-    // takes in a key/value pair array and replces any key found in the string with the value in the array
-    // input
-    // my cool string with {key1} things to replace inside {key2} it
-    // {key1} : value1
-    // {key2} : value2
-    // output
-    // my cool string with value1 things to replace inside value2 it
-    replaceVarsInValue(value, replaceVars) {
```

Comment:
I consolidated the helper methods in Opent2tHelper.js.

---
In reply to: [105346909](https://github.com/openT2T/opent2t-cli/pull/28#discussion_r105346909) [](ancestors = 105346909)